p2p_wallet/Scenes/CreateOrRestoreWallet/CreateWallet/CreateSecurityKeys/CreateSecurityKeys.RootView.swift

```diff
@@ -31,11 +31,16 @@ extension CreateSecurityKeys {
         private let verifyManualButton: WLStepButton = WLStepButton.sub(text: L10n.verifyManually)
         
         private let keysView: KeysView = KeysView()
-        private let keysViewAction: KeysViewActions = KeysViewActions()
+        private let keysViewActions: KeysViewActions = KeysViewActions()
+        private let agreeTermsAndConditions = AgreeTermsAndConditionsView()
```

Comment:
![image](https://user-images.githubusercontent.com/6975538/142641668-201c8229-bd78-4ff1-923f-e54fdd3c5f86.png)
AgreeTermsAndConditionsView: file not found